pkg/server/sandbox_run.go

```diff
@@ -499,6 +552,10 @@ func (c *criService) setupSandboxFiles(id string, config *runtime.PodSandboxConf
 		if err := c.os.Mount("shm", sandboxDevShm, "tmpfs", uintptr(unix.MS_NOEXEC|unix.MS_NOSUID|unix.MS_NODEV), shmproperty); err != nil {
 			return errors.Wrap(err, "failed to mount sandbox shm")
 		}
+		uid := int(UsernsMapping.HostID)
+		if err := os.Chown(sandboxDevShm, uid, uid); err != nil {
```

Comment:
Missing gid, I guess?

Reply:
Right. It works because of how I hardcoded the values in pkg/server/runtime_config_info.go but I should change that to avoid forgetting to add the gid there.